core/src/main/java/io/github/fourlastor/game/demo/round/step/BlobAbsorb.kt

```diff
@@ -63,15 +66,20 @@ class BlobAbsorb @AssistedInject constructor(
         )
     }
 
-    private fun doAttackAnimation(originalPosition: Vector2?, targetPosition: Vector2, continuation: Runnable?) {
+    private fun doAttackAnimation(originalPosition: Vector2, targetPosition: Vector2, continuation: Runnable?) {
         // Distance between source and target is used to scale the animation if needed.
         val distance = source.actorPosition.dst(targetPosition)
         // Angle offset of target from source.
-        val rotationDegrees = calculateAngle(originalPosition, targetPosition)
+        val rotationDegrees = originalPosition.calculateAngle(targetPosition)
         val attackAnimation = Actions.sequence(
             setupAttackAnimation(distance, rotationDegrees), // Move the target unit to the source's Tile.
             Actions.run { targetMon.hex.set(source.hex) },
-            Actions.run { targetMon.actorPosition = originalPosition!! },
+            Actions.run { targetMon.actorPosition = originalPosition }!!,
```

Comment:
is the `!!` required here?

Reply:
I don't understand !! yet so I'll try without it